chapter2/tests/t3/chapter2-t3-velero-eks-diff.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Task 3 - find namespaces without the backup
+set -eo pipefail
+
+readonly VELERO_URL='https://gist.github.com/dmitry-mightydevops/016139747b6cefdc94160607f95ede74/raw/velero.yaml'
```

Comment:
BIG_NAMES are from 90s. So prefer lowecase

Reply:
> BIG_NAMES are from 90s. So prefer lowecase

@dmitry-mightydevops 
I was just following the style guide I was given.
https://google.github.io/styleguide/shellguide.html
 Alright, I'll start writing constants in lowercase as well.